scripts/test/kubernetes-setup.sh

```diff
@@ -5,8 +5,28 @@
 set -x
 set -e
 
-pip install --force-reinstall 'dbt-postgres>=1.8'
-pip install --force-reinstall dbt-adapters
+DBT_VERSION="$1"
+echo "DBT_VERSION:"
+echo "$DBT_VERSION"
+NEXT_MINOR_VERSION=$(echo "$DBT_VERSION" | awk -F. '{print $1"."$2+1}')
+
+pip uninstall dbt-adapters dbt-common dbt-core dbt-extractor dbt-postgres dbt-semantic-interfaces -y
+pip install -U \
+  "dbt-core==$DBT_VERSION" \
+  "dbt-postgres>=$DBT_VERSION,<$NEXT_MINOR_VERSION"
```

Comment:
I am slightly worried on whether they always release a corresponding adapter version. E.g. I don't see yet dbt-postgres 1.10 corresponding to dbt-core 1.10

And this may cause problems when we add 1.10 to the dbt matrix

Reply:
I managed to fix this, to an extent, and solve most of the new issues that popped up. There was only one of the tests that were still failing and I disabled it and created a follow up ticket:
https://github.com/astronomer/astronomer-cosmos/issues/1817